src-electron/db/query-loader.js

```diff
@@ -1124,14 +1124,14 @@ async function insertDeviceTypeFeatures(db, dtClusterRefDataPairs) {
     let dtClusterRef = dtClusterRefDataPair.dtClusterRef
     let clusterData = dtClusterRefDataPair.clusterData
     if ('features' in clusterData && clusterData.features.length > 0) {
-      clusterData.features.forEach((featureCode) => {
-        features.push([dtClusterRef, featureCode])
+      clusterData.features.forEach((feature) => {
+        features.push([dtClusterRef, feature.name, feature.conformance])
```

Comment:
Did feature code and name get mixed up here somehow?

Reply:
<feature code="" name="LT">
The xml put feature code into the name attribute